tests/scenarios/template-tag-codemod-test.ts

```diff
@@ -6,7 +6,7 @@ import { join } from 'path';
 const { module: Qmodule, test } = QUnit;
 
 appScenarios
-  .only('release')
+  .only('lts_5_12')
```

Comment:
This is [failing](https://github.com/embroider-build/embroider/actions/runs/12254779776/job/34186557218#step:5:30) with `release`, not sure why? cc @void-mAlex